app/bundles/PageBundle/EventListener/LeadSubscriber.php

```diff
@@ -144,6 +147,59 @@ public function onTimelineGenerate(LeadTimelineEvent $event)
     }
 
     /**
+     * Compile events for the lead timeline
+     *
+     * @param LeadTimelineEvent $event
+     */
+    public function onTimelineGenerateVideo(LeadTimelineEvent $event)
```

Comment:
This needs to be converted to the new timeline code . https://github.com/mautic/developer-documentation/pull/21/files